internal/builder/config.go

```diff
@@ -89,7 +90,17 @@ func DefaultConfig() Config {
 }
 
 // Validate checks whether the current configuration is valid
-func (c Config) Validate() error {
+func (c *Config) Validate() error {
+	// #nosec G204
```

Comment:
any way to get around having to disable G204 or is the environment the code deemed safe enough that we're not worries about command injections?

Reply:
This parameter is provided by the user, and the code is running on the user's machine, so, the user would be attacking itself. We have plans on building a web UI for this feature, but even for that, I would assume that the Go binary used for building the distribution would be a pre-defined list, or a static value configured by the owner of the service.